docs/NewOBOFC.md

```diff
@@ -21,3 +21,8 @@ Potential members of the Operations Committee are identified from two possible s
 
 Note that new members are strongly encouraged to become members of one of the subcommittees: [Editorial Working Group](https://obofoundry.org/docs/EditorialWG.html)
  or [Technical Working Group](https://obofoundry.org/docs/TechnicalWG.html). 
+
+## Onboarding
+Once a candidate has gone through the steps above and been confirmed as a new OFOC member, they need to onboard.
+Currently, the onboarding steps to be taken by or on behalf of a new OFOC member are listed in a Google doc called "OBO Operations Onboarding" in the OBO Operations gdrive folder.
+This may in the future become a public GitHub doc, if that can be done safely.
```

Comment:
This sentence is not really necessary for our the community to know, I would think? Just drop it and merge the rest!

Reply:
ping @nlharris

Reply:
Sorry, can you clarify what you want to drop? Just line 28? The reason to include that is in case some member of the community challenges us, "Why isn't this onboarding doc public, if you guys are an open organization?"

Reply:
Just line 28!

Reply:
Ok, well, I explained what I see as the purpose of line 28, but if you prefer to drop it, that's fine!

Reply:
Oh ok, I don't mind either way. I just don't think such comments belong in an SOP!

Reply:
What if that comment were in parentheses?

Reply:
Just keep it in if you think it conveys something important:) no worries!